node/src/main/scala/coop/rchain/node/Main.scala

```diff
@@ -108,13 +108,14 @@ object Main {
       case Left(CouldNotConnectToBootstrap) =>
         log.error("Node could not connect to bootstrap node.")
       case Left(InitializationError(msg)) =>
-        log.error(msg)
-        Task.delay(System.exit(-1))
+        log.error(msg) *>
+          Task.delay(System.exit(-1))
```

Comment:
So previously does the System.exit(-1) not run?

Reply:
same question as Kent. as I understand, there is no grantee in ordering with applicatives, so is int possible to exit before logging the error?

Reply:
no, it would be `log.error(msg)` that was ignored
it returned a value (`Task[Unit]`) and that value was discarded

Reply:
`s/*>/>>`